NUAbstractModel.js

```diff
@@ -55,6 +60,8 @@ export default class NUAbstractModel extends NUObject {
                     value = value.toObject();
                 } else if (attributeObj.attributeType === NUAttribute.ATTR_TYPE_ENUM && typeof value === 'object') {
                     value = value.name;
+                } else if (attributeObj.attributeType === NUAttribute.ATTR_TYPE_LIST && attributeObj.subType && typeof attributeObj.subType !== 'string') {
+                    value = value.map(item => item.toObject())
```

Comment:
what about this one? Should you check for null here

Reply:
line 58 already has `if (value)` check, additional check not needed